### ISSUE TITLE
Seg-Fix and OCRnet question

### ISSUE THREAD

**Issue by freaad** [openseg-group/openseg.pytorch#16]
Dear Author

Hello. Thank you for sharing the code about it.
I can get some insight into your code to solve my problem.
I have some questions about your code.
First, in the ocrnet.py, you apply the feature network after that use the OCR block after that you use the F.interpolat 2 times. However, I am wondering why you have two returns about the first interpolation result and the second interpolation result. Usual segmentation network uses the last segmentation as the final segmentation result. 

Second, I want to use your segfix algorithm about my problem. However, I can not find the independent algorithm about it. I also can not find the paper. The readme only mentions that it is similar about the PointRend scheme. I am wondering is it  CNN approach or use the extra others? 
Thank you.

**Comment by PkuRainBow** [comment]
1. 
We do not fully understand your question as we apply the OCR block on the feature map output from the ResNet/stage-4 directly in the ocrnet.py file. We only apply bilinear-upsample on the final predicted logits map, which is the same as the previous DeepLab series and PSPNet series.

> you apply the feature network after that use the OCR block after that you use the F.interpolat 2 times

2.
As for the SegFix paper, we will update the news once release the paper. Please be patient. SegFix is a scheme that is complementary with the PointRend scheme, in fact, our method is related to the Signed-Distance-Transform from principle.

**Comment by freaad** [comment]
https://github.com/openseg-group/openseg.pytorch/blob/6787d215ccc28714b265c48bd9cfc41b41c81e28/lib/models/nets/ocrnet.py#L111

My first question is actually this line. I am wondering why there are two segmentation outputs?

**Comment by PkuRainBow** [comment]
@freaad This design follows the Auxilary Supervision Scheme (or DSN scheme) on the intermediate feature maps output from ResNet-101/Stage3, which is first adopted in PSPNet (Refer to Section 4 of [PSPNet](https://arxiv.org/pdf/1612.01105.pdf) ) in the Semantic Segmentation task.

**Comment by freaad** [comment]
Thank you for your answer.
